src/components/PatronRequestForm/PatronRequestForm.js

```diff
@@ -20,15 +20,26 @@ import { SERVICE_TYPE_COPY, SERVICE_TYPE_LOAN } from '../../constants/serviceTyp
 import { SETTINGS_ENDPOINT } from '../../constants/endpoints';
 
 const PatronRequestForm = ({ autopopulate, copyrightTypes, enabledFields,
-  serviceLevels, currencyCodes, publicationTypes, locations, requesters, onSISelect, operation, patronRequest }) => {
+  serviceLevels, publicationTypes, locations, requesters, tiersByRequester, onSISelect, operation, patronRequest }) => {
   const { change } = useForm();
   const { values } = useFormState();
   const isCopyReq = values?.serviceType?.value === SERVICE_TYPE_COPY;
   const stripes = useStripes();
-
   const CREATE = 'create';
   const EDIT = 'update';
 
+  const currentRequester = values.requestingInstitutionSymbol?.value ?? requesters[0];
+  const tiers = tiersByRequester?.[currentRequester]?.filter(tier => tier.type?.toLowerCase() === values.serviceType?.value) ?? [];
+  const showCost = stripes.config?.reshare?.showCost;
+  const useTiers = stripes.config?.reshare?.useTiers;
+  const resetTier = () => { if (useTiers) change('tier', undefined); };
+  const tier = useTiers && values.tier ? tiers.find(t => t.id === values.tier) : undefined;
+  useEffect(() => {
```

Comment:
Would have been nice if there were a declarative way to do this. Tried setting the `defaultValue` prop on the disabled controls for cost/level but alas it doesn't update the form when changed.